private-templates-service/client/src/components/Common/Tags/Tags.tsx

```diff
@@ -73,9 +76,7 @@ class Tags extends React.Component<Props, State>  {
       if (this.props.tags.includes(tag)) {
         this.highlightTag(tag, this.props.tags);
       }
-      else if (tag === "") {
-        this.closeAddTag();
-      }
+      else if (tag === "") { }
```

Comment:
If condition with an empty body - do we need it or there's another way to organize the logic?

Reply:
done